client/source/components/overview/sub-components/Checkout.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { useEffect, useState } from 'react';
+
+
+const Checkout = function CreateCheckoutComponent({ style }) {
+  const [quantityArray, setQuantityArray] = useState(['SOLD OUT']);
+  const [sizeArray, setSizeArray] = useState(['SOLD OUT']);
+  const [selectedSize, setSelectedSize] = useState('Choose a size...');
+  const [selectedQuantity, setSelectedQuantity] = useState('Choose a quantity...');
+
+  function getSizes(skus) {
```

Comment:
More of a general question. I am noticing a lot of function declarations in this PR. Are those okay now? Or is it just within a component that it's okay?

Reply:
You know the linter was supposed to tell me not to do this. I default that way since it's easier to read while flushing out code... I'm surprised it didn't call me on it!

Per the AirBnB guide, we're not supposed to use them, in favor of function expressions.

Reply:
But then again, reading through it, they are a bit in conflict with themselves. EG:

```
// good
function handleThings(opts = {}) {
  // ...
}```

Reply:
Looks like there's some discussion, example:

https://github.com/airbnb/javascript/commit/019e0f7e07477e85249e1d584f22773120c41b91#r60060792

So just don't use un-named functions being the real goal.

Reply:
You know what? I'm going to rebel! I much prefer the readability of `function foo()` over `const foo = function bar()`, and there's an exisiting rule "no-use-before-define" that keeps you from breaking proper declaration order.

Viva la revolución! ✊